wolfcrypt/src/logging.c

```diff
@@ -118,7 +118,7 @@ THREAD_LS_T void *StackSizeCheck_stackOffsetPointer = 0;
 #ifdef DEBUG_WOLFSSL
 
 /* Set these to default values initially. */
-static wolfSSL_Logging_cb log_function = NULL;
+static wolfSSL_Logging_cb LogFunction = NULL;
```

Comment:
Can you describe the MISRA rule violation for `log_function` vs `LogFunction` for a global variable name? Is it because you renamed the argument in `wolfSSL_SetLoggingCb` and then it conflicted (shadowed global)?